features/step_definitions/seed.rb

```diff
@@ -3,16 +3,16 @@
     school = School.create(:name => 'School of Engineering & Applied Science')
     department = Department.create(:school => school, :name => 'Computer Science')
     subdepartment = Subdepartment.create(:name => 'Computer Science', :mnemonic => 'CS')
-    lastest_semester = Semester.create(:number => 1178, :season => 'Fall', :year => 2017)
-    older_semester = Semester.create(:number => 1172, :season => 'Spring', :year => 2017)
+    lastest_semester = Semester.create(:number => 1178, :season => 'Spring', :year => 2018)
+    older_semester = Semester.create(:number => 1172, :season => 'Fall', :year => 2017)
```

Comment:
You'll want to update these numbers as well, though I don't think it'll matter for the tests.